scripts/exglobal_prep_ocean_obs.py

```diff
@@ -94,9 +136,8 @@ def run_netcdf_to_ioda(obsspace_to_convert):
                 outputFilename = f"gdas.t{cyc}z.{obs_space_name}.{PDY}{cyc}.nc4"
```

Comment:
This is the format of the IODA files expected by the later tasks

Reply:
@AndrewEichmann-NOAA, yes and the output file name is based on that.